android/build.gradle

```diff
@@ -131,6 +136,6 @@ dependencies {
   // noinspection GradleDynamicVersion
   api 'com.facebook.react:react-native:+'
   implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-  implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.1"
+  implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:1.3.3"
   implementation 'io.primer:android:2.0.0'
```

Comment:
Is this ok? Or does it need `SNAPSHOT`?

Reply:
The release version will be `2.0.0`, so it's finr